tests/test_qnode_legacy.py

```diff
@@ -793,6 +793,7 @@ class TestShots:
     """Unit tests for specifying shots per call."""
 
     # pylint: disable=unexpected-keyword-arg
+    @pytest.mark.xfail(reason="deprecated. To be removed in 0.44")
```

Comment:
same as above. Need to continue having this work and test that it works.

Reply:
https://github.com/PennyLaneAI/pennylane/pull/7906/commits/2a164a6041a63f3c6083e9997cd399e6044c51bd